Misc/NEWS.d/next/Library/2019-04-09-22-40-52.bpo-36575.Vg_p92.rst

```diff
@@ -0,0 +1,3 @@
+``_lsprof`` module now uses ``_PyTime_GetPerfCounter`` private API for
+default timer. It will use better timer depending on platform.  Patch by
+Inada Naoki.
```

Comment:
Technically, _PyTime_GetPerfCounter() is the same than time.perf_counter(). Maybe mention time.perf_counter() which is public, rather than mentioning a private method? You may explain that time.perf_counter() is monotonic, whereas previously _lsprof uses time.time() which isn't monotonic.

"It will use better timer depending on platform." well, you can explain that the new implementation now has internally nanosecond resolution, whereas the old implementation of the default timer only has a ressolution of 1 microsecond.

I suggest:

The ``_lsprof`` module now supports internally nanosecond resolution for timing. It now uses time.perf_counter() rather than time.time() (use ``_PyTime_GetPerfCounter`` private API which has nanosecond resolution, rather than gettimeofday() which has microsecond resolution). Timings are no more impact by system clock updates, since perf_counter() is monotonic. Patch by Inada Naoki.